ovirtlib4/ovirtlib.py

```diff
@@ -49,15 +61,18 @@ def __init__(self, host, password, username=defaults.ADMIN_USERNAME, ca_file=Non
         self._events = events.Events(**params)
         self._mac_pools = mac_pools.MacPools(**params)
 
-    def connect(self, *args, **kwargs):
-        """ Made the HTTP connection to remote Engine """
+    def connect(self):
+        """
+        Connects to the remote oVirt instance.
+        """
         return ovirtsdk4.Connection(
             url=self.url,
             username=self.username,
             password=self.password,
             insecure=self.insecure,
             ca_file=self.ca_file,
-            *args, **kwargs
+            log=self.logger,
+            debug=self.debug
```

Comment:
you don't need this, just add *args, **kwargs to the contractor above def __init__()
in this way you support any SDK parameter even future parameters:)